compiler/circlechef/circle/src/Op/BCQFullyConnected.cpp

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "BCQFullyConnected.h"
+
+#include "Convert.h"
+
+namespace circlechef
+{
+
+void CircleOpBCQFullyConnected::filler(const circle::Operator *op, CircleImport *import,
+                                       circlechef::ModelRecipe *model_recipe) const
+{
+  // Nothing to do with filler
```

Comment:
.recipe file in the draft has
```
operand {
  name: "w_a"
  type: FLOAT32
  shape { dim: 32 }
  filler {
    tag: "gaussian"
    arg: "0.0"
    arg: "1.0"
  }
}
```
and seems we need to do something in this method.